screens/CreateGame.js

```diff
@@ -96,21 +142,23 @@ function CreateGame({ navigation }) {
   }
 
   //saves game player that represents each and every game and player association
-  async function storeGamePlayers(gameId, invitedPlayers) {
+  async function storeGamePlayers(gameId) {
     setLoading(true);
```

Comment:
No need to setLoading(true) here since if we reach the point where we call this function, setLoading() must be true already (from create()).